swift/extractor/SwiftDispatcher.h

```diff
@@ -18,8 +20,18 @@ namespace codeql {
 class SwiftDispatcher {
  public:
   // sourceManager, arena, and trap are supposed to outlive the SwiftDispatcher
```

Comment:
you could also update it to a catch-all
```suggestion
  // all references passed as parameters to this constructor are supposed to outlive the SwiftDispatcher
```
so that it does not need to be updated (unless we start passing a reference that is needed only during construction) :slightly_smiling_face: